### PR TITLE
add brightness/contrast settings

### PR DESCRIPTION
**Pull request by stohrendorf**

### CODE REVIEW

File: share/shaders/fx_brightness_contrast.frag [hunk 1]
```diff
@@ -0,0 +1,27 @@
+// based on https://www.shadertoy.com/view/XsjSzR
+
+#include "flat_pipeline_interface.glsl"
+#include "fx_input.glsl"
+#include "util.glsl"
+#include "constants.glsl"
+
+const float Brightness = BRIGHTNESS/100.0/2.0;
+const float Contrast = CONTRAST/100.0/4.0;
```
Comment (serishema) [inline share/shaders/fx_brightness_contrast.frag:9]:
```suggestion
const float Contrast = -CONTRAST/100.0/4.0;
```
Inverting the sign of `CONTRAST` should make it behave the same as brightness, i.e. higher values make the screen brighter.

**Review by serishema — CHANGES_REQUESTED** [review]
The sign of the contrast value is reversed. i.e positive values make the screen darker. 
Could you make it so positive values make it brighter to behave the same as the brightness slider?

File: share/shaders/fx_brightness_contrast.frag (hunk 1 above)
Comment (stohrendorf) [reply share/shaders/fx_brightness_contrast.frag:9]: I agree it's not intuitive, and there should not be such a noticable change in brightness, but inverting the sign would mean that increasing the contrast value in the UI would effectively _decrease_ the contrast. I'll do some more investigation with the formulas.

Comment (stohrendorf) [reply share/shaders/fx_brightness_contrast.frag]: I changed the implementation based on ImageMagick, which now gives better results and also results in a faster GPU calculation. About the brightness change, that's actually valid - if you change the `Range` value in the shader to `1.0` and change the contrast to `-100`, the resulting image changes to full grayscale, so the visual change in brightness is just a result of the loss of color.

**Review by serishema — APPROVED** [review]

**Pull request merged**